libmpv/qthelper.hpp

```diff
@@ -97,8 +97,8 @@ static inline QVariant node_to_variant(const mpv_node *node)
 }
 
 struct node_builder {
-    node_builder(const QVariant& v) {
-        set(&node_, v);
+    node_builder(const QVariant& v, bool force_list = false) {
```

Comment:
It would seem I cannot reproduce what I was experiencing last week. I'll fix these things by next week. Thanks for the feedback.